docs/markdown/Writing Plugins/common-plugin-tasks/plugins-lint-goal.md

```diff
@@ -6,99 +6,103 @@ hidden: false
 createdAt: "2020-07-01T04:51:55.583Z"
```

Comment:
This page incorrectly was a copy of the `fmt` goal. I reverted it to 2.10's `lint` goal, and then fixed it to use the new Options API and to use the arch-aware `Platform`.